<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMachine</class>
 <widget class="QDialog" name="CreateMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateMachine</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Machine Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newMachineName"/>
     </item>
     <item>
      <widget class="QCheckBox" name="configureNowCheckBox">
       <property name="text">
        <string>Configure now</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createPushButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
